packages/toolchain-go/build.yaml

```diff
@@ -1,11 +1,19 @@
 {{ if .Values.variant }}
-image: "golang:{{.Values.version}}-{{.Values.variant}}"
+image: "golang:{{.Values.tag}}-{{.Values.variant}}"
 {{ else }}
-image: "golang:{{.Values.version}}"
+image: "golang:{{.Values.tag}}"
 {{ end }}
 
-{{ if eq .Values.variant "alpine" }}
 prelude:
+{{ if eq .Values.variant "alpine" }}
 - apk update
 - apk add curl make git bash
+{{ end }}
+{{ if eq .Values.variant "bookworm" }}
```

Comment:
:+1: 

unbelievelable, only bookworm doesn't have upx. It is such an historic package - I'm a bit shocked :)

https://packages.debian.org/search?searchon=names&keywords=upx

Reply:
yup I was a bit confused there